demo/deploy-multicluster-osm-ca-bundle.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
This looks like a hack? Let's not deploy root CA certs like this (via a script). We shouldn't be hardcoding certificates in demos.

Reply:
since we need the root certificate to be the same for all the OSM instances participating in the multicluster the osm-ca-bundle needs to be shared. I updated the demo to copy the ca bundle from one cluster to another rather than hardcoding the certificate.